suzieq/poller/worker/nodes/node.py

```diff
@@ -1490,6 +1490,11 @@ async def _init_ssh(self, init_dev_data=True,
         authenticated, not just the main SSH connection.
         '''
         backoff_period = 1
```

Comment:
Can we also fix the commit message please? We're increasing connection backoff time, not reducing it